units/systemd-resolved.service.in

```diff
@@ -13,6 +13,7 @@ Documentation=man:systemd-resolved.service(8)
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/resolved
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/writing-network-configuration-managers
 Documentation=https://www.freedesktop.org/wiki/Software/systemd/writing-resolver-clients
+DefaultDependencies=no
 After=systemd-networkd.service network.target
```

Comment:
if we move this to early boot, we also need to the following changes:

```
After=systemd-sysusers.service
```
(otherwise the systemd-resolved user might not exist yet)

```
Before=shutdown.target
Conflicts=shutdown.target
```

(so that we are shut down at shutdown)